### PR TITLE
docs: minor edits and fix broken links

### PR DESCRIPTION
**Pull request by fviolette**
This PR fixes a few broken links and word case.
I'm adding comments for changes which need specific review.

### CODE REVIEW

File: docs/content/en/use-cases/spread-insert.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Spread `INSERT`s among cluster shards
+title: Spread INSERTs among cluster shards
```
Comment (fviolette) [inline docs/content/en/use-cases/spread-insert.md:2]: Could keep it if was reflected on the website but the backticks propage to `og:title` meta.

File: docs/content/en/use-cases/spread-select.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Spread `SELECT`s from reporting apps among cluster nodes
+title: Spread SELECTs from reporting apps among cluster nodes
```
Comment (fviolette) [inline docs/content/en/use-cases/spread-select.md:2]: Could keep it if was reflected on the website but the backticks propage to `og:title` meta.

File: docs/content/en/install.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ position: 102
 ### Precompiled binaries
 
 Precompiled `chproxy` binaries are available [here](https://github.com/ContentSquare/chproxy/releases).
-Just download the latest stable binary, unpack and run it with the desired [config](#configuration):
+Just download the latest stable binary, unpack and run it with the desired [config](/configuration/default):
```
Comment (fviolette) [inline docs/content/en/install.md:10]: This fixes the broken link but I'm unsure whether this is intended target page, as it is the last of its `category` on the website.

File: README.md [hunk 4]
```diff
@@ -4,6 +4,6 @@
 
 # chproxy
 
-Chproxy is an http proxy and load balancer for [ClickHouse](https://ClickHouse.yandex) database. 
+Chproxy is an HTTP proxy and load balancer for the [ClickHouse](https://ClickHouse.yandex) database.
```
Comment (fviolette) [inline README.md:7]:
```suggestion
Chproxy is an HTTP proxy and load balancer for [ClickHouse](https://ClickHouse.yandex).
```
Actually this suggestion is more consistent with https://github.com/ContentSquare/chproxy/pull/145/files#diff-4887c6195f633baa80b7eefd1b33ca781a4621ce9e9e6b45c9ae99153f1dc5b4R9

File: docs/content/en/install.md (hunk 3 above)
Comment (pixelastic) [reply docs/content/en/install.md:10]: I think that's the correct link; other links explain each individual parts of the config; this one links to the default config

File: docs/content/en/use-cases/spread-insert.md (hunk 1 above)
Comment (pixelastic) [reply docs/content/en/use-cases/spread-insert.md:2]: Hmm... backticks shouldn't propagate to the og:title, we should fix that. Not sure we can fix it in our repo, maybe we'll need to file a PR upstream. I'll add an issue about this.

**Review by pixelastic — APPROVED** [review]
Thanks for the fixes. Nice chinese fixes!

**Pull request merged**